x-pack/test/functional/apps/maps/import_geojson/add_layer_import_panel.js

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import path from 'path';
+
+export default function ({ getPageObjects }) {
+  const PageObjects = getPageObjects(['maps', 'common']);
+
+  const IMPORT_FILE_PREVIEW_NAME = 'Import File';
+  const FILE_LOAD_DIR = 'test_upload_files';
+  const DEFAULT_LOAD_FILE_NAME = 'point.json';
+  const GEO_POINT = 'geo_point';
+  const GEO_SHAPE = 'geo_shape';
+
+  describe('Import layer panel', () => {
+    before(async () => {
+      await PageObjects.maps.openNewMap();
+    });
+
+    beforeEach(async () => {
+      await PageObjects.maps.clickAddLayer();
+      const panelOpen = await PageObjects.maps.isLayerAddPanelOpen();
```

Comment:
This check can be removed. The test will fail if the panel does not open and the error message will be easier to debug if it can not find a selector vs having "expected true to be false"